File: BECore/app/model/Customer.js
/// <summary>
/// Model EXTJS Customer.
/// </summary>
Ext.define('QLDT.model.Customer', {
    extend: 'Ext.data.Model',
    idProperty: 'CustomerID',
    idType: 'Guid',
    fields: [
        { name: "CustomerID", type: "int" },
        { name: "CustomerCode", type: "string" },
        { name: "CustomerName", type: "string" },
        { name: "CustomerGender", type: "int" },
        { name: "DOB", type: "date" },
        { name: "HealthCareNumber", type: "string" },
        { name: "Tel", type: "string" },
        { name: "Email", type: "string" },
        { name: "Address", type: "string" },
        { name: "CustomerGroup", type: "string" },
        { name: "CustomerDescription", type: "string" },
        { name: "PresenterName", type: "string" },
        { name: "PresenterPhone", type: "string" },
        { name: "PresenterAddress", type: "string" },
        { name: "PresenterIDC", type: "string" },
        { name: "Relationship", type: "string" },
        { name: "CreatedDate", type: "date" },
        { name: "ModifiedDate", type: "date" },
        { name: "IPAddress", type: "string" },
        { name: "ModifiedBy", type: "string" },
        { name: "CreatedBy", type: "string" },
    ]
});

File: MT.Model/HTTTYT/Customer.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace MT.Model
{
    #region Customer:EntityBase
    /// <summary>
    /// This object represents the properties and methods of a Customer:EntityBase.
    /// </summary>
    public class Customer : EntityBase
    {
        [Key]
        public int CustomerID { get; set; }
        public string CustomerCode { get; set; }
        public string CustomerName { get; set; }
        public int CustomerGender { get; set; }
        public DateTime DOB { get; set; }
        public string HealthCareNumber { get; set; }
        public string Tel { get; set; }
        public string Email { get; set; }
        public string Address { get; set; }
        public string CustomerGroup { get; set; }
        public string CustomerDescription { get; set; }
        public string PresenterName { get; set; }
        public string PresenterPhone { get; set; }
        public string PresenterAddress { get; set; }
        public string PresenterIDC { get; set; }
        public string Relationship { get; set; }


    }
    #endregion
}

